app/models/zip_part.rb

```diff
@@ -21,7 +21,7 @@ class ArchivePreservedCopyPart < ApplicationRecord
   # @return [Boolean] true if all expected parts are now replicated
```

Comment:
i might be missing something, but to me the suggested comment seems duplicative of the existing comment (`For this persisted part, are it and all its cohort now replicated (to one endpoint)?`).

could we edit the current comment, or remove it in favor of the new one?

FWIW, i actually find the old one clearer (because it says "all its cohort", implying that it's only about the one `ArchivePreservedCopy` to which the given part belongs, whereas the suggested comment seems ambiguous, and reads to me like it could be about all the `ArchivePreservedCopy`s having all their parts replicated).

either way, i find the presence of both comments to be extra confusing.

Reply:
Ok by me, thank you!